redfish-core/lib/ethernet.hpp

```diff
@@ -840,6 +923,95 @@ inline void createIPv6(const std::string& ifaceId, uint8_t prefixLength,
         "");
 }
 
+/**
+ * @brief Deletes given Static Route
+ *
+ * @param[in] ifaceId     Id of interface whose IP should be deleted
+ * @param[in] ipHash      DBus Hash id of IP that should be deleted
+ * @param[io] asyncResp   Response object that will be returned to client
+ *
+ * @return None
+ */
+inline void
+    deleteStaticRoute(const std::string& ifaceId, const std::string& ipHash,
+                      const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec) {
+        if (ec)
+        {
+            messages::internalError(asyncResp->res);
+        }
+        },
+        "xyz.openbmc_project.Network",
+        "/xyz/openbmc_project/network/" + ifaceId + ipHash,
+        "xyz.openbmc_project.Object.Delete", "Delete");
+}
+
+/**
+ * @brief Deletes the static route entry for this interface and creates a
+ * replacement static route entry
+ *
+ * @param[in] ifaceId      Id of interface upon which to create the static route
+ * entry
+ * @param[in] destination  Destination address of static route
+ * entry
+ * @param[in] prefixLength Prefix for the subnet mask
```

Comment:
Can you switch the order here so these are in order?

Reply:
done